config/filesystems.php

```diff
@@ -69,6 +69,16 @@
             'url' => env('AWS_URL'),
         ],
 
+        'minio' => [
+            'driver' => 's3',
+            'key' => env('MINIO_ACCESS_KEY_ID'),
+            'secret' => env('MINIO_SECRET_ACCESS_KEY'),
+            'region' => env('MINIO_DEFAULT_REGION'),
+            'bucket' => env('MINIO_BUCKET'),
+            'use_path_style_endpoint' => true,
+            'endpoint' => env('MINIO_ENDPOINT', 'http://127.0.0.1:9005'),
+        ],
```

Comment:
This should be default Laravel configuration,
https://github.com/laravel/laravel/tree/8.x/config

The remaining developer can add as per their need.